### PR TITLE
Pasting changes from release-1.0 for README and docs structure.

### PR DESCRIPTION
**Pull request by TheKLARKEN**
This adjusts the README and docs links for 1.0.0 (but for 2.0 in the future). The docs site now points to the 1.0.0 release and will follow release versioning for the immediate future. This is a short term adjustment to get the install and usage docs up to date for 2.0 dev, but the processes and docs are expected to drastically change in the future. If you reviewed my other PR, this one is exactly the same.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,30 @@
...
 
-Supergiant implements simple practices that abstract load-balancing, application deployment, basic monitoring, node deployment or destruction, and more, on a highly usable UI. Its efficient packing algorithm enables seamless auto-scaling of Kubernetes clusters, minimizing costs while maintaining the resiliency of applications. To dive into top-level concepts, see [the documentation](https://supergiant.readthedocs.io/en/docs/About%20Supergiant/architecture/).
+Supergiant implements simple practices that abstract load-balancing, application deployment, basic monitoring, node deployment or destruction, and more, on a highly usable UI. Its efficient packing algorithm enables seamless auto-scaling of Kubernetes clusters, minimizing costs while maintaining the resiliency of applications. To dive into top-level concepts, see [the documentation](https://supergiant.readthedocs.io/en/v1.0.0/About%20Supergiant/architecture/).
```
Comment (stgleb) [inline README.md]: Maybe it is better to use word compaction, but I am not sure.

Comment (TheKLARKEN) [reply README.md]: As in "Its efficient packing algorithm..." becomes "Its efficient compaction algorithm...?"

Comment (stgleb) [reply README.md]: Yeah, when you say packing I usually think about packaging like dockerizing, wraping something into something. But compaction is more about making usage of resource smaller. Like compaction in garbage collection that fixes memory fragmentation and usage.

Comment (TheKLARKEN) [reply README.md]: I like it, I'll change it. 🙂

Comment (TheKLARKEN) [reply README.md]: @stgleb It is now adjusted and some other things are fixed!

**Pull request merged**